build/musicbrainz/scripts/start.sh

```diff
@@ -15,5 +15,5 @@ then
   cron -f &
 fi
 
-/start_mb_renderer.pl
+dockerize -wait tcp://db:5432 -timeout 60s -wait tcp://mq:5672 -timeout 60s -wait tcp://redis:6379 -timeout 60s /start_mb_renderer.pl
```

Comment:
What happens with ef8d1a9063a48fa26ed43f84d78c30b6bb822706 if replication cron starts before `db` is available?

Reply:
Then move it to https://github.com/metabrainz/musicbrainz-docker/blob/ef8d1a9063a48fa26ed43f84d78c30b6bb822706/build/musicbrainz/scripts/start.sh#L11 ? I kinda want a kick start with the resource compiling that the rest is up and we don't waste time waiting.

Reply:
Sounds more reliable.